database/dataGeneration/Cassandra/index.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const cassandra = require('cassandra-driver');
+
+// COPY watches (id, wid, watch_name, series, size, watch_price) FROM '/Users/miles/Desktop/Hack_Reactor/sdc/MVMT-summary/database/dataGeneration/seedWatchData1.csv' CSV;
```

Comment:
In your next pull request, you may want to delete this commented out former (?) code!